google/datalab/contrib/mltoolbox/commands/_ml.py

```diff
@@ -410,7 +414,19 @@ def _transform(args, cell):
   if '--project-id' not in cmd_args:
     cmd_args.extend(['--project-id', google.datalab.Context.default().project_id])
 
-  _shell_process.run_and_monitor(cmd_args, os.getpid(), cwd=MLTOOLBOX_CODE_PATH)
+  try:
+    tmpdir = None
```

Comment:
This works and I am okay with that. Is it possible to not delete the temp dir? For example, use a hash of the package URL as the dir name, so every time we delete the files in the dir first before downloading the package into the same dir. The benefits:

1. If user clicks "Reset" button, the temp dir will be cleaned up next time so at most we have one dir per package to keep on disk.

2. We don't need the try/finally blocks here.